pages/common/nix.md

```diff
@@ -5,3 +5,19 @@
 - Search for a package via its name or description:
 
 `nix search {{search_term}}`
+
+- Start a nix shell with the specified packages available:
+
+`nix run {{nixpkgs.pkg_names}}`
```

Comment:
It's unclear from this syntax how one would provide multiple packages. Could we rework this example to make that clearer?

Reply:
Perhaps `nix run {{nixpkgs.pkg1}} {{nixpkgs.pkg2}} {{nixpkgs.pkg3...}}`?

The `nixpkgs` string indicates that the the packages must be prefixed by the "channel" they come from (almost always `nixpkgs` for non-advanced users).

Reply:
We can do {{nixpkgs.pkg1 nixpkgs.pkg2 nixpkgs.pkg3…}}

Note that … is an unicode ellipsis.

Reply:
Weird, the comment box must have done that. Would it be okay to commit it I replace the unicoded ellipsis with 3 periods?

Reply:
I agree it makes more sense to wrap the entire list in the token markers, rather than having multiple separate tokens with an ellipsis inside the last one.

As for the Unicode ellipsis, it seems to me that @agnivade was suggesting to use the Unicode one rather than three ASCII periods. However, that kinda goes against #2802 (/cc @urbalazs), so I would vote for using a regular ASCII ellipsis. That said, I don't have a strong opinion on the matter.

Reply:
@mebeim recently made a commit which changed all 3 ASCII periods to unicode ellipsis. I don't have much expertise on this matter. Just want things to be consistent.

Reply:
In that case I assume it would make most sense to comply with the rest of the project? If unicode is a problem then it should be handled for the entire repository, not just this file.

Reply:
Well, #2802 did not edit those (which got in earlier, in #2745), so my guess is that the intention was not to remove all non-ASCII characters altogether, but focus on particularly confusable / easily missed ones.

On the other hand, I'm still wary for the same reason I argued against requiring trailing slashes in URLs [here](https://github.com/tldr-pages/tldr/pull/2684#issuecomment-452132239): it seems to me that people are more likely to use regular ellipsis than Unicode ones when authoring pages, so we'd have to regularly go through all pages to find inconsistencies (or create automated checks).

Just food for thought -- if you guys still prefer Unicode ellipsis, I won't insist further (in this thread 😜)

Reply:
Yeah, I don't mind either way. @sbrl and @mebeim were both in favor of adding unicode ellipsis and I was neutral.

Guys, what are your thoughts on @waldyrious' reasoning ?